api/openapi.yml

```diff
@@ -112,6 +112,8 @@ paths:
         - name: windowSize
           in: query
           required: false
+          description: |
+            If not specified, a single usage aggregate will be returned for the entirety of the specified period for each subject and group.
```

Comment:
I think a separate endpoint would be more devex-friendly. But maybe an SDK can have multiple methods and abstract the logic away.